src/app_service/service/privacy/service.nim

```diff
@@ -84,6 +85,7 @@ QtObject:
 
       let isPasswordOk = self.accountsService.verifyAccountPassword(defaultAccount, password)
       if not isPasswordOk:
+        data.errorMsg = "Incorrect current password"
```

Comment:
@borismelnik please take a look at [verifyAccountPassword](https://github.com/status-im/status-desktop/blob/c0202df0cbd786ed8d3ba8977c33d515958d78a0/src/app_service/service/accounts/service.nim#L345-L356), in case of an error, maybe we should display an error received from the `status-go` instead of this one? What do you think?

One of solution may be to update `verifyAccountPassword` to return `string` instead of bool and in case of an empty string we can consider that all passed well, otherwise we can display that error received from the `status-go`.